docs/snap-clone.md

```diff
@@ -92,3 +99,106 @@ csi-cephfs-pvc       Bound    pvc-1ea51547-a88b-4ab0-8b4a-812caeaf025d   1Gi
 cephfs-pvc-clone     Bound    pvc-b575bc35-d521-4c41-b4f9-1d733cd28fdf   1Gi        RWX            csi-cephfs-sc  39s
 cephfs-pvc-restore   Bound    pvc-95308c75-6c93-4928-a551-6b5137192209   1Gi        RWX            csi-cephfs-sc  55m
 ```
+
+# Create RBD snapshot and Clone Volume
+
+Before continuing, make sure you enabled the required
+feature gate `VolumeSnapshotDataSource=true` in your Kubernetes cluster.
+
+In the `examples/rbd` directory you will find two files related to snapshots:
+[snapshotclass.yaml](./rbd/snapshotclass.yaml) and
+[snapshot.yaml](./rbd/snapshot.yaml).
```

Comment:
These links don't seem to work for me

Reply:
Okay, let me check. I have mentioned as it was mentioned in README.md